docs/portal/image-registry/viewing-logs.md

```diff
@@ -2,12 +2,10 @@
 
 ECP keeps a log of your registry projects. The log contains a date- and time-stamped record of the operations performed by registry users. The following procedure describes how to view logs. To make sure you view the logs for the appropriate project, you can view a project's information before viewing its logs.
 
-**Note** Users with project update permission can edit this page. Users without project update permission can only read this page.
-
 1. In the left pane, click **Image Registry > Registry Projects**.
 2. Perform one of the following tasks:<ul><li>Click a project's **Actions** button and select **Logs**.</ul></li> <ul><li>To view information about a project before viewing its logs, click its name to display a details page. To view the project's logs, click the **Logs** button at the top right.</ul></li>
 The Access Logs page lists the log entries associated with the selected project. The most current entries appear at the top of the list. Filters at the top of the page allow you to filter entries by registry user, repository, operation, tag, and time range.<br>
 **Note:** The more filters you use, the narrower the results. For example, if you enter criteria for User, Repository, and Operation, the log records must match all three sets of criteria to appear in the log list. Records that match one or two criteria are not shown.</br>
-   ![null](</docs/resources/images/registry/log-project.png>)
+<p align=center><img src="/docs/resources/images/registry/log-project.png" width="600"></p>
 3. Buttons and fields at the bottom right allow you to view other pages and increase the number of entries shown on a page.
 4. When you finish viewing, click the **Cancel** button at the bottom right.
```

Comment:
On page preview, this step 4 is displayed immediately after step 3 without a line break in-between. 
Besides, \*\*Cancel\*\* is not rendered properly as a bold text.

Reply:
Fixed.